parquet-column/src/main/java/parquet/vector/ColumnVector.java

```diff
@@ -97,4 +101,23 @@ protected void initDecoder(DataPage page) throws IOException {
     else
       decoder.initFromPage(page.getValueCount(), ((DataPageV2)page).getData().toByteArray(), 0);
   }
+
+  public static final ColumnVector createVector(ColumnDescriptor descriptor) {
+    switch (descriptor.getType()) {
+      case DOUBLE:
+        return new DoubleColumnVector();
+      case FLOAT:
+        return new FloatColumnVector();
+      case INT32:
+        return new IntColumnVector();
+      case INT64:
+        return new LongColumnVector();
+      case INT96:
+//      case BINARY:
+//      case FIXED_LEN_BYTE_ARRAY:
+//        return new ByteColumnVector();
```

Comment:
commented this out as the ``ByteColumnVector`` constructor requires an ``isLazy`` argument. Need to sort this out.